src/controls/Controls.spec.js

```diff
@@ -1 +1,19 @@
 // Test away!
+import React from 'react';
+import { render, fireEvent } from 'react-testing-library';
+import renderer from 'react-test-renderer';
+
+import Controls from './Controls';
+
+describe('<Controls />', () => {
+  it('Should render', () => {
+    const { getByText } = render(<Controls />);
+  });
+
+  it('Should have a "Close Gate" & "Lock Gate" button by default', () => {
+    const { getByText } = render(<Controls />);
+
+    getByText(/close gate/i);
+    getByText(/lock gate/i);
```

Comment:
If we wanted to generalize our helper for use with Controls as well, we could do so pretty easily:
```javascript
const makeRenderer = Component => {
  return (texts = [], props = {}) => () => {
    const { getByText } = render(<Component {...props} />);

    texts.forEach(text => {
      getByText(text);
    });
  };
};
```
and, the usage:
```javascript
const renderControls = makeRenderer(Controls);

// ...
it('Should have a "Close Gate" & "Lock Gate" button by default',
  renderControls([ /close gate/i, /lock gate/i ])
);
```